.github/workflows/acceptance_tests_secondary.yml

```diff
@@ -8,6 +8,8 @@ on:
       - '.github/workflows/acceptance_tests_secondary.yml'
       - '.github/workflows/acceptance_tests_common.yml'
       - '!java/*.changes*'
+  schedule:
+    - cron: '0 6 * * *'
```

Comment:
LGTM. I just wonder how we can monitor these results later, as I guess they will not be part of any PR. They will be store somewhere in the master branch tabs 👀